data_steward/validation/main.py

```diff
@@ -1001,17 +1001,6 @@ def write_drc_pii_validation_file():
     return consts.DRC_VALIDATION_REPORT_SUCCESS
 
 
-@api_util.auth_required_cron
-@log_traceback
-def write_sites_pii_validation_files():
-    project = bq_utils.app_identity.get_application_id()
-    validation_dataset = bq_utils.get_validation_results_dataset_id()
-    logging.info(f"Calling write_results_to_site_buckets")
-    matching.write_results_to_site_buckets(project, validation_dataset)
-
-    return consts.SITES_VALIDATION_REPORT_SUCCESS
```

Comment:
Need to remove this from the constants file too.

Reply:
Removed!